src/Raven.Server/Documents/AI/Embeddings/EmbeddingsHelper.cs

```diff
@@ -24,28 +25,31 @@ public static class EmbeddingsHelper
     /// </remarks>
     internal static readonly List<string> ValuesListToVerifyConnection = ["TestValue", "TestValue2"];
 
+    [SkipLocalsInit]
     public static string CalculateInputValueHash(string value)
     {
+        // Here we take a string, and we:
+        // * convert to upper case
+        // * trim all white space
+        // * generate a hash
+        //
+        // The idea is that we want to try (as much as possible) to be able
+        // to get better hash values, so "keyboard" and "Keyboard " would end up
+        // being the same. Note that we are being pretty simplistic here on purpose,
+        // to avoid having a complex rule set for "tokenization" these, in addition to
+        // the actual tokens
+        Span<char> buffer = stackalloc char[256];
+        if(value.Length > buffer.Length)
+            buffer = new char[value.Length];
+        var len = value.AsSpan().Trim().ToUpperInvariant(buffer);
```

Comment:
Why do we prefer upper case over lower case? Is there any practical reason? Does it matter?

Reply:
No reason, no